<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Sebastian Trueg</author>
 <class>base_K3bAudioCdTextAllFieldsWidget</class>
 <widget class="QWidget" name="base_K3bAudioCdTextAllFieldsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>269</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="8" column="0">
    <widget class="QLabel" name="textLabel8">
     <property name="text">
      <string>Messa&amp;ge:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>m_editMessage</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout">
     <item>
      <widget class="KLineEdit" name="m_editSongwriter"/>
     </item>
     <item>
      <widget class="QToolButton" name="m_buttonCopySongwriter">
       <property name="toolTip">
        <string>Copy to all tracks</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="textLabel6">
     <property name="text">
      <string>&amp;UPC EAN:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>m_editUpc_ean</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout">
     <item>
      <widget class="KLineEdit" name="m_editComposer"/>
     </item>
     <item>
      <widget class="QToolButton" name="m_buttonCopyComposer">
       <property name="toolTip">
        <string>Copy to all tracks</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="textLabel4">
     <property name="text">
      <string>So&amp;ngwriter:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>m_editSongwriter</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="textLabel5">
     <property name="text">
      <string>&amp;Composer:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>m_editComposer</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="textLabel7">
     <property name="text">
      <string>&amp;Disk id:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>m_editDisc_id</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout">
     <item>
      <widget class="KLineEdit" name="m_editArranger"/>
     </item>
     <item>
      <widget class="QToolButton" name="m_buttonCopyArranger">
       <property name="toolTip">
        <string>Copy to all tracks</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="textLabel3">
     <property name="text">
      <string>&amp;Arranger:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>m_editArranger</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="KLineEdit" name="m_editUpc_ean"/>
   </item>
   <item row="7" column="1">
    <widget class="KLineEdit" name="m_editDisc_id"/>
   </item>
   <item row="8" column="1">
    <widget class="KLineEdit" name="m_editMessage"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="textLabel2">
     <property name="text">
      <string>&amp;Performer:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>m_editPerformer</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout">
     <item>
      <widget class="KLineEdit" name="m_editTitle"/>
     </item>
     <item>
      <widget class="QToolButton" name="m_buttonCopyTitle">
       <property name="toolTip">
        <string>Copy to all tracks</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>&amp;Title:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>m_editTitle</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="Line" name="line4">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout">
     <item>
      <widget class="KLineEdit" name="m_editPerformer"/>
     </item>
     <item>
      <widget class="QToolButton" name="m_buttonCopyPerformer">
       <property name="toolTip">
        <string>Copy to all tracks</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_editTitle</tabstop>
  <tabstop>m_buttonCopyTitle</tabstop>
  <tabstop>m_editPerformer</tabstop>
  <tabstop>m_buttonCopyPerformer</tabstop>
  <tabstop>m_editArranger</tabstop>
  <tabstop>m_buttonCopyArranger</tabstop>
  <tabstop>m_editSongwriter</tabstop>
  <tabstop>m_buttonCopySongwriter</tabstop>
  <tabstop>m_editComposer</tabstop>
  <tabstop>m_buttonCopyComposer</tabstop>
  <tabstop>m_editUpc_ean</tabstop>
  <tabstop>m_editDisc_id</tabstop>
  <tabstop>m_editMessage</tabstop>
 </tabstops>
 <includes>
  <include location="local">klineedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
